### ISSUE TITLE
Lite Version Doesn't work on Ubuntu 22.04

### ISSUE THREAD

**Issue by rany2** [TheAssassin/AppImageLauncher#508]
### Pre-submit checks

- [X] I [checked for similar issues](https://github.com/TheAssassin/AppImageLauncher/issues?q=) beforehand, but could not find any, not even closed ones. I could not add my bug report to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Describe the bug

AppimageLauncher doesn't start up due the service it installs encountering the following error:
```
/tmp/.mount_appimatujU5N/usr/bin/appimagelauncherd: symbol lookup error: /lib/x86_64-linux-gnu/libpango-1.0.so.0: undefined symbol: g_memdup2
```

This happens on the AppImage variant (deb not tested).

### Expected behavior

Just starts up

### Distribution and desktop environment

Ubuntu 22.04 

### Installed AppImageLauncher version

`appimagelauncher-lite-2.2.0-travis89-a1a3c90-x86_64.AppImage`

**Comment by TheAssassin** [comment]
Please try the latest continuous pre-release build.

**Comment by rany2** [comment]
> Please try the latest continuous pre-release build.

It is the latest one.

**Comment by TheAssassin** [comment]
Nope, https://github.com/TheAssassin/AppImageLauncher/releases/tag/continuous is the latest pre-release.

**Comment by rany2** [comment]
Where should I download this from? Could you just provide a link?

**Comment by TheAssassin** [comment]
Sorry, my bad. We used to build binaries on Travis with old release (it's been a while...), and all new artifacts _should_ carry `gha` (GitHub actions) instead of `travis`. Indeed, this is the latest binary, there's a bug that causes the broken name somewhere.

**Comment by rany2** [comment]
Ah, no problem!

**Comment by mikew** [comment]
Also seeing this issue on a Arch (Steam Deck)

**Comment by danstewart** [comment]
Same on Fedora 36 using the latest build: `appimagelauncher-lite-2.2.0-gha100-5a0fdec-x86_64.AppImage`

**Comment by TheAssassin** [comment]
I'll be rebuilding continuous Lite AppImages, the issue should be fixed then.

**Comment by TheAssassin** [comment]
Confirmed fixed on lubuntu/xubuntu 22.04
